Firestore/Swift/Source/Codable/SelfDocumentId.swift

```diff
@@ -16,32 +16,37 @@
 
 import FirebaseFirestore
 
-/// A special type used to mark a custom object field to be automatically
-/// populated with the document's referecne when the custom object is created
-/// from a Cloud Firestore document (for example, via
-/// `DocumentSnapshot.data()`).
+/// A value that is populated in Codable objects with a `DocumentReference` by
+/// the FirestoreDecoder when a document is read.
+///
+/// Note that limitations in Swift-compiler generated Codable implementations
```

Comment:
Now that I'm seeing this in context "Swift-compiler generated" looks weird. It should either be "Swift compiler-generated" or just "Swift compiler generated". Sorry.

Reply:
Done.